src/xmipp/bindings/python/envs_DLTK/xmipp_pyTorch-gpu.yml

```diff
@@ -9,10 +9,12 @@ channels:
   - nvidia
 dependencies:
   - python=3.10
-  - numpy=1.23
   - mrcfile=1.4.3
-  - kornia=0.6.8  #The last that is provided as *.bz2 extension
-  - starfile=0.4.11 #The last that that is provided as *.bz2 extension
   - pytorch==1.13.1
-  - pytorch-cuda=11.7
   - torchvision=0.14
+  - pytorch-cuda=11.7
+  - pip  
+  - pip:
+      - numpy==1.23 
+      - kornia==0.6.8    #The last that is provided as *.bz2 extension
+      - starfile==0.4.11  #The last that that is provided as *.bz2 extension
```

Comment:
With pip we don't have this restriction. Remove the comment and relax the version restriction